week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,7 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var total = numberOfStudents + numberOfMentors;
```

Comment:
Again, minor thing, but renaming this variable might make it clearer what it represents :)